src/components/Receiving/ReceivingList.js

```diff
@@ -63,6 +64,7 @@ class ReceivingList extends Component {
         query: '',
       },
     },
+    receive: RECEIVE,
```

Comment:
it makes not too much sense to declare it here if all you want - to pass it below. Just declare it in ItemDetails

Reply:
Anyway I should pass `mutator` to `ItemDetails`

Reply:
What is the reason to pass mutator there and not to declare it there? It tightly couples components and if all you want - to pass `receive` or change _path - it makes not too much sense in passing.